File: easy/1480-running-sum-of-1d-array/solution.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
      arr1=[]
      arr1.append(nums[0])
      for i in range(1,len(nums)):
        x=nums[i]+arr1[i-1]
        arr1.append(x)  

      return arr1

File: easy/1480-running-sum-of-1d-array/solution.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int>arr1;
        arr1.push_back(nums[0]);
        for (int i=1;i<nums.size();i++){
            nums[i]+=nums[i-1];
            arr1.push_back(nums[i]);
        }
        return arr1;
    }
};